docs/Building-Transmission.md

```diff
@@ -39,7 +39,7 @@ ninja -C build transmission-gtk
 ## On Unix ##
 ### Prerequisites ###
 
-#### Debian 11 / Bullseye ####
+#### Debian 12 / Bookworm ####
```

Comment:
I like the idea of including 12, but this does lose the fact that one can build on 11 this way... so how about:

```suggestion
#### Debian 11 and Newer ####
```